src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/ManagedInstanceProtectorTests.ps1

```diff
@@ -0,0 +1,61 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+$mangedInstanceRg = "MlAndzic_RG"
```

Comment:
same here, all resources should be created per test and names should be randomly generated using getAssetName

Reply:
Addressed in my comment below